spanner_dbapi/cursor.py

```diff
@@ -168,7 +170,7 @@ def __do_execute_insert_homogenous(self, transaction, parts):
         return transaction.insert(table, columns, values)
 
     def __handle_DQL(self, sql, params):
-        with self._connection.read_snapshot() as snapshot:
+        with self._connection.snapshot() as snapshot:
```

Comment:
I do not see the definition of the `snapshot` method in the `Connection` class either.